web-wallet/src/routes/(app)/dashboard/migrate/+page.svelte

```diff
@@ -18,4 +18,8 @@
   });
 </script>
 
-<MigrateContract />
+{#await networkStore.getNetworkName() then name}
+  {#if import.meta.env.VITE_FEATURE_MIGRATE === "true" && ["mainnet", "testnet"].includes(name)}
```

Comment:
I wouldn't repeat the feature flag check here if we have it in the contract descriptors.